lib/matplotlib/tests/test_cbook.py

```diff
@@ -938,3 +940,45 @@ def test_auto_format_str(fmt, value, result):
     """Apply *value* to the format string *fmt*."""
     assert cbook._auto_format_str(fmt, value) == result
     assert cbook._auto_format_str(fmt, np.float64(value)) == result
+
+
+def test_unpack_to_numpy_from_torch():
+    """Test that torch tensors are converted to numpy arrays.
+    We don't want to create a dependency on torch in the test suite, so we mock it.
+    """
+    class Tensor:
+        def __init__(self, data):
+            self.data = data
+        def __array__(self):
+            return self.data
+    torch = ModuleType('torch')
+    torch.Tensor = Tensor
+    sys.modules['torch'] = torch
+
+    data = np.arange(10)
+    torch_tensor = torch.Tensor(data)
+
+    result = cbook._unpack_to_numpy(torch_tensor)
+    assert isinstance(result, np.ndarray)
```

Comment:
I would be more specific: `_unpack_to_numpy` should not just return any array, but exercise the `__array__` interface
```suggestion
    assert result is torch_tensor.__array__()
```

Note that formally, we would want
```
    nptest.assert_array_equal(_unpack_to_numpy(torch.Tensor(data)), data)
```
which should work for any array wrapper, but since we're mocking anyway I think it's justifiable to test the mocked implementation detail.

Reply:
I agree!